packages/kbn-profiling-utils/common/flamegraph.ts

```diff
@@ -84,6 +84,18 @@ export interface ElasticFlameGraph
  * @returns ElasticFlameGraph
  */
 export function createFlameGraph(base: BaseFlameGraph): ElasticFlameGraph {
+  // This loop jumps over the error frames in the graph.
+  // This code is temporary until we decided how to present error frames in the UI.
+  // Error frames only appear as child nodes of the root frame.
+  // Error frames only have a single child node.
+  for (let i = 0; i < base.Edges[0].length; i++) {
+    const childNodeID = base.Edges[0][i];
+    // eslint-disable-next-line no-bitwise
```

Comment:
You can just cast to number (`as number`) first before the bit operation -- this way you don't have to disable the linter.